delorean-user-data.txt

```diff
@@ -280,18 +280,19 @@ write_files:
     permissions: 0755
 
 bootcmd:
-#  - echo "" >> /etc/ssh/sshd_config
-#  - echo "Port 330" >> /etc/ssh/sshd_config
-#  - systemctl restart sshd
+  - echo "" >> /etc/ssh/sshd_config
```

Comment:
Appending to sshd_config like this on each boot generates errors in the log but looks like sshd still runs